float64 distance
float64[3] rotational_vector
float64[3] translational_vector
uint32 id
float64 yaw
nav_msgs/Odometry odom